docs/docs/guides/web3_eth_migration_guide.md

```diff
@@ -0,0 +1,1098 @@
+# Web3Eth Migration Guide
+
+## Breaking Changes
+
+### Not Implemented or Exported
+
+-   [extend](https://web3js.readthedocs.io/en/v1.7.3/web3-eth.html#extend) functionality not implemented
+-   [web3.eth.createAccessList](https://web3js.readthedocs.io/en/v1.7.3/web3-eth.html#createaccesslist) not implemented
+-   [web3.eth.personal](https://web3js.readthedocs.io/en/v1.7.3/web3-eth.html#personal) namespace is not exported
+-   [web3.eth.net](https://web3js.readthedocs.io/en/v1.7.3/web3-eth.html#net) namespace is not exported
+
+### Defaults and Configs
+
+-   `givenProvider` default value is `undefined` instead of `null`
+-   `currentProvider` will never return `null`, provider required upon instantiation as opposed to being optional in 1.x //todo maybe this is not true after after the recent discussion/change
+-   `web3.eth.defaultAccount` default value is `undefined` instead of `null`
```

Comment:
That applies to everywhere. So insted of listing it for individual values, we should highlight it overall. 

All the API level interfaces returning or accepting `null` are replaced with `undefined`

Reply:
is extended implemented too?

Reply:
Regarding point 2. Sound good to me.
Let's wait for input from the rest of the team for their opinion and to see how they have structured docs.